packages/services/src/competition.service.ts

```diff
@@ -231,7 +233,6 @@ type CompetitionDetailsData = {
       agentId: string | null;
     }>;
     votingEnabled?: boolean;
-    openForBoosting: boolean;
```

Comment:
That simplification I was talking about to relieve some of the typing craziness we have going on.